source/common/http/conn_manager_impl.cc

```diff
@@ -789,7 +789,7 @@ void ConnectionManagerImpl::startDrainSequence() {
 void ConnectionManagerImpl::ActiveStream::refreshCachedRoute() {
   Router::RouteConstSharedPtr route = snapped_route_config_->route(*request_headers_, stream_id_);
   request_info_.route_entry_ = route ? route->routeEntry() : nullptr;
-  cached_route_.value(std::move(route));
+  cached_route_ = (std::move(route));
```

Comment:
Can we get rid of the extra parens at sites like this?

Reply:
I saw this coming ;). will update these things as well.